emerging_optimizers/orthogonalized_optimizers/orthogonalized_optimizer.py

```diff
@@ -154,7 +158,12 @@ def step(self, closure: Callable[[], float] | None = None) -> float | None:
                 if group["weight_decay"] > 0.0:
                     if group["use_decoupled_weight_decay"]:
                         # Apply decoupled weight decay
```

Comment:
FIX: comment needs to be updated or removed as they don't say anything more than code